packages/vats/src/localchain.js

```diff
@@ -5,6 +5,8 @@ import { AmountShape } from '@agoric/ertp';
 
 const { Fail, bare } = assert;
 
+/** @import {TypedJson, ResponseType, Proto3Shape} from '@agoric/cosmic-proto'; */
```

Comment:
```suggestion
/** @import {TypedJson, Proto3Shape} from '@agoric/cosmic-proto'; */
```

Reply:
i wouldn't worry about extra types for now. especially because ambients can make it look like imports that aren't used that actually are during build time